python-3.10/Dockerfile

```diff
@@ -113,6 +115,7 @@ RUN mkdir $VIRTUAL_ENV \
         charset-normalizer\<3 \
         cryptography\<41 \
         pytz\<2023 \
+        'pyopenssl==23.1.1' \
```

Comment:
![Screenshot 2024-08-16 at 13 45 19](https://github.com/user-attachments/assets/fd54054d-8103-47a2-8438-f04d312d31b3)

Tady jsem to musel zafixovat. Nevim proc je tam podminka pro ten `cryptography` balicek, ale chci to nechat co nejvic stejne jak to bylo

Reply:
Taky mne to napadlo :D